src/cli.ts

```diff
@@ -51,6 +57,10 @@ const cli = meow(
       --recurse, -r
           Recursively follow links on the same root domain.
 
+      --retry,
```

Comment:
This is probably a naive question, but: does this mean we need to pass in `--retry`, or `--retry true` to use this?

Thanks again, Justin!

Reply:
Both should work!

Reply:
👍